### PR TITLE
the fixes was made

### PR DESCRIPTION
**Pull request by wlima-dev**
All mentioned points that was require is fixed now.

### CODE REVIEW

File: src/main/java/com/ecommerce/wlm_ecommerce_product_service/domain/event/OrderCreatedEvent.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.ecommerce.wlm_ecommerce_product_service.domain.event;
+
+public class OrderCreatedEvent {
```
Comment (yuraf640) [inline src/main/java/com/ecommerce/wlm_ecommerce_product_service/domain/event/OrderCreatedEvent.java]: Why you need this?

File: src/main/java/com/ecommerce/wlm_ecommerce_product_service/domain/exception/ProductWasNotUpdated.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.ecommerce.wlm_ecommerce_product_service.domain.exception;
+
+public class ProductWasNotUpdated extends RuntimeException {
```
Comment (yuraf640) [inline src/main/java/com/ecommerce/wlm_ecommerce_product_service/domain/exception/ProductWasNotUpdated.java]: Usually a custom exceptions must be created to be catch and some additional work can be done. If you extend RuntimeException - it is not checked exception and it will not be catched. So i don't see a reason why you need this exception at all

File: src/main/java/com/ecommerce/wlm_ecommerce_product_service/domain/service/TaxesCalculator.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package com.ecommerce.wlm_ecommerce_product_service.domain.service;
+
+public class TaxesCalculator {
```
Comment (yuraf640) [inline src/main/java/com/ecommerce/wlm_ecommerce_product_service/domain/service/TaxesCalculator.java]: Why you need this?

File: src/main/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/config/KafkaConfig.java [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package com.ecommerce.wlm_ecommerce_product_service.infrastructure.config;
+
+public class KafkaConfig {
```
Comment (yuraf640) [inline src/main/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/config/KafkaConfig.java]: Why you need this?

File: src/main/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/messaging/OrderCreatedProducer.java [hunk 5]
```diff
@@ -0,0 +1,4 @@
+package com.ecommerce.wlm_ecommerce_product_service.infrastructure.messaging;
+
+public class OrderCreatedProducer {
```
Comment (yuraf640) [inline src/main/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/messaging/OrderCreatedProducer.java]: Why you need this?

File: src/main/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/messaging/OrderStatusConsumer.java [hunk 6]
```diff
@@ -0,0 +1,4 @@
+package com.ecommerce.wlm_ecommerce_product_service.infrastructure.messaging;
+
+public class OrderStatusConsumer {
```
Comment (yuraf640) [inline src/main/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/messaging/OrderStatusConsumer.java]: Why you need this?

File: src/test/java/com/ecommerce/wlm_ecommerce_product_service/WlmEcommerceProductServiceApplicationTests.java [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class WlmEcommerceProductServiceApplicationTests {
+
+	@Test
+	void contextLoads() {
+	}
```
Comment (yuraf640) [inline src/test/java/com/ecommerce/wlm_ecommerce_product_service/WlmEcommerceProductServiceApplicationTests.java]: Why you need this? If you have some test it must not be empty.

File: src/test/java/com/ecommerce/wlm_ecommerce_product_service/application/dto/OrderDTOTest.java [hunk 8]
```diff
@@ -0,0 +1,4 @@
+package com.ecommerce.wlm_ecommerce_product_service.application.dto;
+
+public class OrderDTOTest {
```
Comment (yuraf640) [inline src/test/java/com/ecommerce/wlm_ecommerce_product_service/application/dto/OrderDTOTest.java]: Why you need this? If you have some test it must not be empty.

File: src/test/java/com/ecommerce/wlm_ecommerce_product_service/application/dto/UserDTOTest.java [hunk 9]
```diff
@@ -0,0 +1,4 @@
+package com.ecommerce.wlm_ecommerce_product_service.application.dto;
+
+public class UserDTOTest {
```
Comment (yuraf640) [inline src/test/java/com/ecommerce/wlm_ecommerce_product_service/application/dto/UserDTOTest.java]: Why you need this? If you have some test it must not be empty.

File: src/test/java/com/ecommerce/wlm_ecommerce_product_service/application/service/OrderServiceTest.java [hunk 10]
```diff
@@ -0,0 +1,181 @@
...
+                () -> orderService.getOrderById(99L));
+
+        verify(orderRepository, times(1)).findOrderById(99L);
+    }
+
+
+    @Test
+    @DisplayName("Should return Order if id exists")
+    void shouldGetOrderById() {
+//        Order expectedOrder = new Order(1L, 10L, List.of());
```
Comment (yuraf640) [inline src/test/java/com/ecommerce/wlm_ecommerce_product_service/application/service/OrderServiceTest.java]: please add content for this test

File: src/test/java/com/ecommerce/wlm_ecommerce_product_service/domain/model/UserTest.java [hunk 11]
```diff
@@ -0,0 +1,5 @@
+package com.ecommerce.wlm_ecommerce_product_service.domain.model;
+
+public class UserTest {
+
```
Comment (yuraf640) [inline src/test/java/com/ecommerce/wlm_ecommerce_product_service/domain/model/UserTest.java]: Why you need this? If you have some test it must not be empty.

File: src/test/java/com/ecommerce/wlm_ecommerce_product_service/domain/repository/UserRepositoryTest.java [hunk 12]
```diff
@@ -0,0 +1,4 @@
+package com.ecommerce.wlm_ecommerce_product_service.domain.repository;
+
+public class UserRepositoryTest {
```
Comment (yuraf640) [inline src/test/java/com/ecommerce/wlm_ecommerce_product_service/domain/repository/UserRepositoryTest.java]: Why you need this? If you have some test it must not be empty.

File: src/test/java/com/ecommerce/wlm_ecommerce_product_service/domain/service/TaxesCalculatorTest.java [hunk 13]
```diff
@@ -0,0 +1,4 @@
+package com.ecommerce.wlm_ecommerce_product_service.domain.service;
+
+public class TaxesCalculatorTest {
+}
```
Comment (yuraf640) [inline src/test/java/com/ecommerce/wlm_ecommerce_product_service/domain/service/TaxesCalculatorTest.java]: Why you need this? If you have some test it must not be empty.

File: src/test/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/config/BeanConfigTest.java [hunk 14]
```diff
@@ -0,0 +1,4 @@
+package com.ecommerce.wlm_ecommerce_product_service.infrastructure.config;
+
```
Comment (yuraf640) [inline src/test/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/config/BeanConfigTest.java]: Why you need this? If you have some test it must not be empty.

File: src/test/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/messaging/MessageProducerTest.java [hunk 15]
```diff
@@ -0,0 +1,4 @@
+package com.ecommerce.wlm_ecommerce_product_service.infrastructure.messaging;
+
+public class MessageProducerTest {
```
Comment (yuraf640) [inline src/test/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/messaging/MessageProducerTest.java]: Why you need this? If you have some test it must not be empty.

File: src/test/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/persistence/adapter/OrderRepositoryImplTest.java [hunk 16]
```diff
@@ -0,0 +1,4 @@
+package com.ecommerce.wlm_ecommerce_product_service.infrastructure.persistence.adapter;
+
+public class OrderRepositoryImplTest {
+}
```
Comment (yuraf640) [inline src/test/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/persistence/adapter/OrderRepositoryImplTest.java]: Why you need this? If you have some test it must not be empty.

File: src/test/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/persistence/entity/ProductEntityTest.java [hunk 17]
```diff
@@ -0,0 +1,4 @@
+package com.ecommerce.wlm_ecommerce_product_service.infrastructure.persistence.entity;
+
+public class ProductEntityTest {
+}
```
Comment (yuraf640) [inline src/test/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/persistence/entity/ProductEntityTest.java]: Why you need this? If you have some test it must not be empty.

File: src/test/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/persistence/repository/SpringDataProductRepositoryTest.java [hunk 18]
```diff
@@ -0,0 +1,4 @@
+package com.ecommerce.wlm_ecommerce_product_service.infrastructure.persistence.repository;
+
+public class SpringDataProductRepositoryTest {
```
Comment (yuraf640) [inline src/test/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/persistence/repository/SpringDataProductRepositoryTest.java]: Why you need this? If you have some test it must not be empty.

File: src/test/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/rest/ProductControllerTest.java [hunk 19]
```diff
@@ -0,0 +1,4 @@
+package com.ecommerce.wlm_ecommerce_product_service.infrastructure.rest;
+
+public class ProductControllerTest {
```
Comment (yuraf640) [inline src/test/java/com/ecommerce/wlm_ecommerce_product_service/infrastructure/rest/ProductControllerTest.java]: Why you need this? If you have some test it must not be empty.

**Review by yuraf640 — CHANGES_REQUESTED** [review]
Please remove all the trash and fix the tests. Be careful: if you create a pull request, it means you're ready to merge the code and release it. Your code looks good, but it's not ready for release due to the large amount of trash.

**Review by yuraf640 — APPROVED** [review]
